bool has_arrived	#if true, the robotino has arrived 
